index.js

```diff
@@ -18,6 +19,12 @@ updateNotifier({pkg}).notify()
 main(parseArgs())
 
 function main (argv) {
+  const shell = argv['shell-auto-fallback']
+  if (shell || shell === '') {
```

Comment:
doesn't having a default mean that you always get `''` if you don't provide the option, though? I'm so confused.